nautobot/project-static/css/base.css

```diff
@@ -299,6 +299,7 @@ table.report th a {
 }
 .panel table {
     margin-bottom: 0;
+    overflow: hidden;
```

Comment:
Did you refer to the previous discussion on https://github.com/nautobot/nautobot/pull/5929#discussion_r1679542993? This change may impact *a lot* of panel tables, so please be sure to manually test in a lot of different places (object list, related object tables in detail view, home page, etc.) especially with small window sizes to make sure this global change is correct/desirable in all cases.

Reply:
Pulled this down locally and it looks like it fixes the square-corner problem in Firefox. It doesn't fix other problems like the table overflow described in https://github.com/nautobot/nautobot/pull/5929#discussion_r1679631833 - it looks like the issue with the BulkDelete views on narrow screens is because they aren't wrapping the table in a `<div class="panel panel-default table-responsive">` but only in a `<div class="panel panel-default">`. We probably need to audit our usage of panels containing object tables in general and make sure they're all correctly flagged as responsive tables? But I'm fine with us doing that in a separate issue/PR.